### PR TITLE
Cog 6 setup project

### PR DESCRIPTION
**Pull request by kelfink**
Implements Docker and docker compose stupor the project.

### CODE REVIEW

File: docker/web/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM ruby:2.4.2
```
Comment (RamuKammagani-osi) [inline docker/web/Dockerfile]: Please update this to ruby 2.5.1

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        target: /app/app
+    ports:
+      - ${COUNTY_ADMIN_APP_PORT}:3000
+      - ${COUNTY_ADMIN_APP_WEBPACK_PORT}:3035
+    environment:
+      SECRET_KEY_BASE: ${SECRET_KEY_BASE}
+      COUNTY_ADMIN_API_BASE_URL: "http://${DEV_IP}:${CASE_API_PORT}/"
+      COUNTY_ADMIN_APP_JS_API_URL: ${COUNTY_ADMIN_APP_JS_API_URL}
+      PERRY_API_BASE_URL: "http://${DEV_IP}:${PERRY_PORT}/perry"
+      GEO_API_BASE_URL: "http://${DEV_IP}:${GEO_API_PORT}/"
```
Comment (RamuKammagani-osi) [inline docker-compose.yml]: I guess we don't need Geo Services at all

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      AUTHENTICATION: ${AUTHENTICATION}
+      FEATURE_DASHBOARD_CASE_ACTIONS: ${FEATURE_DASHBOARD_CASE_ACTIONS}
```
[the remark this answers is not in the document]
Comment (RamuKammagani-osi) [reply docker-compose.yml]: +1

File: docker/web/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+WORKDIR /app
+COPY Gemfile Gemfile.lock /app/
+RUN bundle install --jobs 20 --retry 5
+COPY package.json yarn.lock /app/
+RUN yarn install --production=false --non-interactive --frozen-lockfile
+COPY . /app
+RUN NODE_ENV=production RAILS_ENV=production bundle exec rails assets:precompile
+RUN yarn install --production=false --non-interactive --frozen-lockfile
+# Required because we have mapping issues in nginx
+RUN ln -s /app/public/casemng/packs /app/public/packs
```
Comment (Sushmareddy-OSI) [inline docker/web/Dockerfile]: /app/public/casemng/packs /app/public/packs, here path says casemng.

**Comment by kelfink** [comment]
In fact, this pull request is for COG-10

**Comment by kelfink** [comment]
retest this please

**Comment by kelfink** [comment]
retest this please

**Comment by kelfink** [comment]
retest this please

File: docker/web/Dockerfile (hunk 1 above)
Comment (kelfink) [reply docker/web/Dockerfile]: done

File: docker/web/Dockerfile (hunk 4 above)
Comment (kelfink) [reply docker/web/Dockerfile]: done

**Pull request merged**